ntbs-service/Models/Validations/ValidationHelper.cs

```diff
@@ -80,8 +80,7 @@ public static string ValidYearLaterThanBirthYear(int birthYear)
         public const string DenotificationDateLatestToday = "Date of denotification cannot be later than today";
         public const string DenotificationReasonRequired = "Please supply a reason for denotification";
         public const string DenotificationReasonOtherRequired = "Please supply additional details for the denotification reason";
-        public const string NotificationDateRequired = "Please supply notification date";

```

Comment:
Is this an accidental delete of notificationDateRequired? Which I think should make it fail the added integration tests?

Reply:
This was the duplicate of the existing error message